iceoryx_utils/test/moduletests/test_file_reader.cpp

```diff
@@ -47,7 +47,10 @@ class FileReader_test : public Test
         internal::CaptureStdout();
 
         std::fstream fs(TestFilePath, std::fstream::out | std::fstream::trunc);
-        fs << TestFileContent;
+        if (fs.std::fstream::is_open())
+        {
+            fs << TestFileContent;
+        }
```

Comment:
please add an `ASSERT_STREQ("expected open fstream", "fstream not open");` in the else branch